File: Retos_1_10/reto_5.py
#Escribe un programa que pida al usuario un número y muestre en pantalla su tabla de multiplicar del 1 al 10.
mult = (1,2,3,4,5,6,7,8,9,10)
x = input("Ingresa un numero: ")
print(f"Tabla del {x}")
for y in mult:
    print(f"{mult[y-1]} * {x} = {int(x)*int(mult[y-1])}")
#la forma mas eficiente es for y in range(1,11), en este caso queria practicar con una tupla

File: Retos_1_10/reto_5.js
//Escribe un programa que pida al usuario un número y muestre en pantalla su tabla de multiplicar del 1 al 10.
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,  
  output: process.stdout 
});
rl.question("Ingresa un número: ", (input) => {
let mult = [1,2,3,4,5,6,7,8,9,10];
console.log(`Tabla del ${input}`)
for (let valor of mult){
    console.log(`${valor} * ${input} = ${input * valor}`)};
rl.close();});
//podria usar forEach o No utilizar ninguna array, for (let i = 1; i <= 10; i++) {console.log(`${numero} x ${i} = ${numero * i}`);}